### ISSUE TITLE
Citation not rendered correctly

### ISSUE THREAD

**Issue by idnsunset** [readthedocs/sphinx_rtd_theme#1030]
### Problem

https://github.com/readthedocs/sphinx_rtd_theme/pull/838 fixes footnotes and glossary issues, but not citation.

#### Error Logs/Results

![2020-12-30-165814_458x341_scrot](https://user-images.githubusercontent.com/31075263/103340995-8e821c80-4ac0-11eb-9b17-f05572cb471d.png)

#### Expected Results

![2020-12-30-170155_428x308_scrot](https://user-images.githubusercontent.com/31075263/103341056-c12c1500-4ac0-11eb-8fb4-482b00ba41ff.png)

### Environment Info

- Python Version: 3.9.1
- Sphinx Version: 3.3.1
- RTD Theme Version: 0.5.0

ref https://github.com/readthedocs/sphinx_rtd_theme/issues/942

**Comment by stsewd** [comment]
Hi, this is already fixed. You are seeing the old theme https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#footnotes, you can see the current theme here https://sphinx-rtd-theme--1039.org.readthedocs.build/en/1039/demo/demo.html#references  (see https://github.com/readthedocs/sphinx_rtd_theme/pull/1039). Let us know if you have a reproducible project if that isn't the case.

**Comment by idnsunset** [comment]
Has this fix been included in the 0.5.1 version from PyPI?

**Comment by JamesRandom** [comment]
> Has this fix been included in the 0.5.1 version from PyPI?

Apparently not: https://james-r-rtd-test.readthedocs.io/en/latest/
![Screenshot 2021-02-28 at 13 52 53](https://user-images.githubusercontent.com/11873471/109419156-5e75cd80-79cc-11eb-8e67-c54794b2ccb1.png)

https://github.com/JamesRandom/rtd-test/blob/4f04a593202cb03786318937e103c3d3dd31db90/requirements.txt#L4

**Comment by stsewd** [comment]
> Has this fix been included in the 0.5.1 version from PyPI?

yes!

**Comment by idnsunset** [comment]
But it's still styled correctly on my machine.

![2021-03-05-172355_738x370_scrot](https://user-images.githubusercontent.com/31075263/110095348-cadf3b00-7dd7-11eb-9d6b-d88bc8ef2da9.png)

```
pip show sphinx-rtd-theme
Name: sphinx-rtd-theme
Version: 0.5.1
Summary: Read the Docs theme for Sphinx
Home-page: https://github.com/readthedocs/sphinx_rtd_theme
Author: Dave Snider, Read the Docs, Inc. & contributors
Author-email: dev@readthedocs.org
License: MIT
Location: /usr/local/lib/python3.9/dist-packages
Requires: sphinx
Required-by:
```
